src/test/java/Homework21.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+class Homework21 extends BaseTest {
+    String expectedRenamePlaylistMsg = "Updated playlist \"My New Playlist.\"";
+    @Test
+    public void renamePlaylist() {
+        provideEmail("merry.lordlie@testpro.io");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        doubleCLickPlaylist();
+        enterNewPlaylist("My New Playlist");
+        Assert.assertEquals(renamePlaylistMsg(), expectedRenamePlaylistMsg);
+    }
+
+    public void enterNewPlaylist(String playlist) {
+        WebElement newPlaylist = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[name='name']")));
+        actions.click(newPlaylist).perform();
+        newPlaylist.sendKeys(Keys.COMMAND, "A", Keys.DELETE);
```

Comment:
Hi 
I have checked your homework 15, but Test script is failed! You got this problem, please correct it.
Best wishes from the HW team (Khanh, khanh@testpro.io)

![image](https://github.com/mlordlie/A49-QA-Automation-Framework/assets/44592718/504035ac-2c1e-4a21-868f-0215554229e7)

In the step click and clear newPlaylist value, you need to use `Keys.CONTROL`  to sendKeys
// HINT: `newPlaylist.sendKeys(Keys.CONTROL, "A", Keys.DELETE);`